app/javascript/components/student/tracks-list/TagOptionList.jsx

```diff
@@ -17,25 +18,32 @@ export function TagOptionList({
 
   return (
     <form onSubmit={onSubmit}>
-      {options.map((option) => {
-        return (
-          <div key={option.category}>
-            <p>{option.category}</p>
-            {option.options.map((option) => {
-              return (
-                <TagOption
-                  key={option.value}
-                  onChange={handleChange}
-                  checked={selectedTags.includes(option.value)}
-                  label={option.label}
-                  value={option.value}
-                />
-              )
-            })}
-          </div>
-        )
-      })}
-      <button>Apply</button>
+      <div className="categories">
+        {options.map((option) => {
+          return (
+            <div key={option.category} className="category">
+              <h4>{option.category}</h4>
```

Comment:
This is good, but probably the incorrect level heading. Probably should be `h2`, having the "invisible" dialog title be `h1`.

Reply:
My understanding is that you should only have one `h1` on a page. Is this a seperate rule that dialogs effectively "reset"? And if so, how does this effect SEO?

Reply:
It's a bit weird. It's based on the fact that I didn't see a single `h1` on the page. I usually use `h2` for dialog titles (so only one h1, and each dialog has an h2). 

You can replace the `aria-label` of this dialog with `aria-labelledby="dialog-header"` and add `<h2 id="dialog-header" class="visually-hidden">the label</h2>`.

That will yield a much better _document outline_ (which is used often by people who use screenreaders). In that case, these _must_ become `h3`.

Find `.visually-hidden` here: https://www.a11yproject.com/posts/2013-01-11-how-to-hide-content/